### ISSUE TITLE
Update text on admin app Upgrades page

### ISSUE THREAD

**Issue by amandacilek** [medic/cht-core#5392]
- [x] In the current version information, change “At” to “On” (“at” is for times, “on” is for dates”)
- [x] At the top, below the current version information but before the upgrades, to explain the difference between stage and install: “To upgrade your application to a specific release, beta, or branch, it is recommended that you stage first. This allows for background work to be done to prepare the installation without interrupting users. Once staging is complete, click Install to proceed with the upgrade. This action cannot be undone, so please make sure your data has been backed up and your users are notified of downtime.”
- [x] In the releases, betas, and branches sections, add title above the first column called “Name”
- [x] In the releases, betas, and branches sections, add title above the date column called “Date”
~In the Releases section, create a new column and add a title above called “Release Notes”~

**Comment by garethbowen** [comment]
> In the Releases section, create a new column and add a title above called “Release Notes”

I decided not to do this point as it's covered by a separate issue: #3995

**Comment by garethbowen** [comment]
> In the current version information, change “At” to “On” (“at” is for times, “on” is for dates”)

Translation modified in POE to be exported before release.

**Comment by garethbowen** [comment]
Ready for AT in `5392-improve-upgrade-page`

**Comment by newtewt** [comment]
Hey just a call out. Seems like the column names are not what's defined above.  I was expecting them to be "Name" and "Date" but they are "Version" and "Release Date". I think that later works better but wanted to verify before closing.

**Comment by newtewt** [comment]
Also, I see a `\` when upgrading now. I pulled the latest from POE with the script.


![image](https://user-images.githubusercontent.com/10531066/67586144-e8e65700-f71e-11e9-823a-32239bde9d8b.png)

**Comment by garethbowen** [comment]
@newtewt It looks fine on my machine. The `\!` is in the default translations file and has been there for some time so I think it's ok.

After you exported the translations how did you import them into the app?

**Comment by newtewt** [comment]
@garethbowen  I exported using `poe export ./translations 3.8.0` which creates blank files. Then exported without `poe export ./translations` and uploaded with medic-conf. I don't see the values. 


![image](https://user-images.githubusercontent.com/10531066/67768667-6664dc80-fa29-11e9-92b4-d27d4df00b75.png)

**Comment by garethbowen** [comment]
@newtewt I've raised a new issue for the escaping issue - it affects other translations too. You can work around this by uploading the translations on the App Settings > Languages > English > Import button.

**Comment by ngaruko** [comment]
I guess we can close this @newtewt ?

**Comment by garethbowen** [comment]
Merged.
